lib/service/database.js

```diff
@@ -66,28 +66,42 @@ var affectedTableRegexCache = {
     pixelHeight: /!pixel_height!/g
 };
 
-DatabaseService.prototype.getLastUpdatedTimeFromAffectedTables = function(node, skip, callback) {
+DatabaseService.prototype.getLastUpdatedTimeAndTableDataFromAffectedTables = function(node, skip, callback) {
```

Comment:
I don't see it like source metadata but I agree with give it a more generic name
Maybe we could call it `getTableDataFromAffectedTables` because the main purpose of this function is to retrieve data from the affected tables found.